src/services/email.service.js

```diff
@@ -0,0 +1,69 @@
+require('dotenv/config');
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
+
+const send = ({ email, subject, html }) => {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+};
+
+const sendActivationEmail = (email, token) => {
+  const href = `${process.env.HOST}/activation/${token}`;
```

Comment:
Ensure that the HOST environment variable is correctly set and includes the protocol (http or https) to avoid broken links in the emails.